packages/office-ui-fabric-react/src/components/Callout/Callout.types.ts

```diff
@@ -172,15 +177,28 @@ export interface ICalloutProps extends React.Props<Callout | CalloutContent> {
    */
   setInitialFocus?: boolean;
 
-  /**
-    * Deprecated at v0.59.1, to be removed at >= v1.0.0. Pass in a beakWidth to dictate size.
-    * @deprecated
-    */
-  beakStyle?: string;
-
   /**
    * Set max height of callout
    * When not set the callout will expand with contents up to the bottom of the screen
    */
   calloutMaxHeight?: number;
+
+  theme?: ITheme;
```

Comment:
Documentation comments?

Reply:
sure...i've got to remove the scss file and update snapshot too